tests/checkout/index.c

```diff
@@ -89,6 +89,65 @@ void test_checkout_index__can_remove_untracked_files(void)
 	cl_assert_equal_i(false, git_path_isdir("./testrepo/dir"));
 }
 
+void test_checkout_index__can_disable_pathspec_match(void)
+{
+	static git_index *index;
+	git_oid commit_id;
+	git_checkout_options g_opts = GIT_CHECKOUT_OPTIONS_INIT;
+	git_object *g_object;
+
+	char *files_to_checkout[] = { "test10.txt", "test11.txt"};
+	size_t files_to_checkout_size = 2;
+
+	/* reset to beginning of history (i.e. just a README file) */
+	g_opts.checkout_strategy =
+		GIT_CHECKOUT_FORCE | GIT_CHECKOUT_REMOVE_UNTRACKED;
+
+	cl_git_pass(git_revparse_single(&g_object, g_repo, "8496071c1b46c854b31185ea97743be6a8774479"));
+	cl_git_pass(git_checkout_tree(g_repo, g_object, &g_opts));
+	cl_git_pass(
+		git_repository_set_head_detached(g_repo, git_object_id(g_object)));
+	git_object_free(g_object);
+	g_object = NULL;
+
+	cl_git_pass(git_repository_index(&index, g_repo));
+
+	/* We create 4 files and commit them */
+	cl_git_mkfile("testrepo/test9.txt", "original\n");
+	cl_git_mkfile("testrepo/test10.txt", "original\n");
+	cl_git_mkfile("testrepo/test11.txt", "original\n");
+	cl_git_mkfile("testrepo/test12.txt", "original\n");
```

Comment:
Is it perhaps just missing the `./testrepo...`. It seems like all the other tests follow that format except for this one.